api/app/util/ApibuilderServiceImportResolver.scala

```diff
@@ -0,0 +1,27 @@
+package util
+
+import db.{Authorization, VersionsDao}
+import io.apibuilder.api.v0.models.Version
+import io.apibuilder.spec.v0.models.Service
+
+object ApibuilderServiceImportResolver {
+
+  def resolveChildren(service: Service, versionsDao: VersionsDao, auth: Authorization): Map[String, Service] = {
+
+    def resolve(service: Service, acc: Map[String, Service] = Map.empty): Map[String, Service] = {
```

Comment:
I think you can annotate this with `@tailrec`

Reply:
it's not a valid tailrec